lib/src/protocol/http/parser/tests.rs

```diff
@@ -1785,14 +1785,14 @@ fn header_cookies_sticky_end() {
         BufferMove::Advance(7),
         BufferMove::Advance(7),
         BufferMove::Delete(1),
-        BufferMove::Delete(18),
+        BufferMove::Delete(16),
```

Comment:
this is very likely a bug. With this serie of moves for the headers `Cookie:BAR=FOO;SOZUBALANCEID=0; \r\nHost: hello.com`, we would send the following slices:
- `Cookie:`
- `BAR=FOO`
- `SOZUBALANCEID=0;` is not transmitted. The two following spaces were trimmed when parsing but are still in the buffer
- `  ` <- the two spaces
- then the parser thinks that the next header starts at `\r\nHost: hello.com`